docs/CHANGELOG.md

```diff
@@ -1,6 +1,60 @@
 BlockTrail PHP SDK Changelog
 ============================
 
+v2.0.0
+------
+ - New [Default] Wallet Version 2
```

Comment:
should probably mention that new wallets support password changing (the main reason we changed it)